code/lib/preview-api/README.md

```diff
@@ -0,0 +1,94 @@
+# Preview (Web)
+
+This is the main API for the (web) version of the Storybook Preview.
+
+The preview's job is:
+
+1. Read and update the URL (via the URL Store)
+
+2. Listen to instructions on the channel and emit events as things occur.
+
+3. Render the current selection to the web view in either story or docs mode.
+
+## V7 Store vs Legacy (V6)
+
+The story store is designed to load stories 'on demand', and will operate in this fashion if the `storyStoreV7` feature is enabled.
```

Comment:
Do we have a unified way of typing "story store"? So is it "Story Store", "story store" or "StoryStore"? Minor, but helps to keep domain understanding.

Reply:
I think "Story Store" although I'm not sure we mention it in prose too much.

@ndelangen maybe we should make a ticket to clean up some of these READMEs.

Reply:
Agreed, This readme, needs a thorough rework, as @tmeasday also points out.

Reply:
I've moved the README files a bit, so they are in 1 place. They'll likely need extensive work to make proper sense again. I was hoping to do that in a follow-up PR..